### ISSUE TITLE
Issues related to data split

### ISSUE THREAD

**Issue by LeoZhiheng** [url-kaist/HeLiMOS-PointCloud-Toolbox#3]
Thank you for your excellent work (HeLiMOS). However, I have some questions about the results in TABLE III of the paper.

I want to know if the training data comes from L+A, are the performance of L, A, O, and V validated on the validation set or the test set?

**Comment by LeoZhiheng** [comment]
We greatly look forward to your response and sincerely thank you.

**Comment by OREOCHIZ** [comment]
Hello, LeoZhiheng! 
Thank you for showing interest in our dataset and for your question. :smiley: 
All our experimental results were conducted on the **test** set.
You can also check this in the HeLiMOS section of the **README.md** for MapMOS or 4DMOS repositories.

MapMOS: https://github.com/PRBonn/MapMOS
4DMOS : https://github.com/PRBonn/4DMOS

**Comment by LimHyungTae** [comment]
If the question has been resolved, could you close this issue? @LeoZhiheng

**Comment by LeoZhiheng** [comment]
> Hello, LeoZhiheng! Thank you for showing interest in our dataset and for your question. 😃 All our experimental results were conducted on the **test** set. You can also check this in the HeLiMOS section of the **README.md** for MapMOS or 4DMOS repositories.
> 
> MapMOS: https://github.com/PRBonn/MapMOS 4DMOS : https://github.com/PRBonn/4DMOS

Thank you very much for your reply!
